### DISCUSSION TITLE
How can I get the logit value corresponding to a specific string after model forward?

### DISCUSSION THREAD

**Discussion by CownowAn** [sgl-project/sglang#1946] [Q&A]
I loaded the reward model, and in Hugging Face, I obtained the scalar value reward using the following code:
```python
reward = self.model(input_ids).logits[0].mean(dim=-1).sigmoid().cpu().tolist()[loc]
```
Here, `loc` is the position of a specific tag string.

How can I perform this in sglang?

**Comment by merrymercy** [comment]
Use `return_logprob=True` and `logprob_start_len=0`
see https://sgl-project.github.io/references/sampling_params.html
